client/src/components/Main/RSVPFloating/index.js

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { theme1, grey3, lightBlue, eventPink, eventGreen } from "../../../resources/style-constants";
+import SecondaryButton from "../../Common/Buttons/SecondaryButton/index.js";
+import PrimaryButton from "../../Common/Buttons/PrimaryButton/index.js";
+import RSVPCard from "../RSVPCard/index.js"
+import styled from "styled-components";
+
+const accentTheme = {
+  default: lightBlue,
```

Comment:
To be consistent with the blues, "link" is the color that we have been using elsewhere.